salt/modules/djangomod.py

```diff
@@ -112,6 +115,70 @@ def syncdb(settings_module,
                    *args, **kwargs)
 
 
+def migrate(settings_module,
+           app_label=None,
+           migration_name=None,
+           bin_env=None,
+           database=None,
+           pythonpath=None,
+           env=None,
+           noinput=True,
+           runas=None):
+    '''
+    Run migrate
+
+    Execute the Django-Admin migrate command (requires Django 1.7 or higher).
+
+    .. versionadded:: Neon
+
+    app_label
+        Specific app to run migrations for, instead of all apps.
+        This may involve running other apps’ migrations too, due to dependencies.
+
+    migration_name
+        Named migration to be applied to a specific app.
+        Brings the database schema to a state where the named migration is applied,
+        but no later migrations in the same app are applied. This may involve
+        unapplying migrations if you have previously migrated past the named migration.
+        Use the name zero to unapply all migrations for an app.
+
+    database
+        Database to migrate. Defaults to 'default'.
+
+    noinput
+        Suppresses all user prompts. Defaults to True.
```

Comment:
can you add docs for all of the options that can be passed in?

Reply:
Sure, the rest are either default options for django-admin (https://docs.djangoproject.com/en/2.1/ref/django-admin/#default-options), or args that get passed along to cmd.run, but I'll add some quick documentation for them.